docs/compendium/substitute.md

```diff
@@ -1,31 +1,107 @@
-# Substitute compendia
+# Substitute two compendia
 
-introduction...
+Substitution is the combination of an base erc and an overlay erc.
+A user can choose files from the overlay erc that will be replaced with files of the base erc or uniquely added.
 
 ## Create substitution
 
 ### Request
 
-`POST /api/v1/..`
+`POST /api/v1/substitution`
+
+input of request-body for substitution
+```json
+{
+  "base": "base_erc_id",
+  "substFiles": [
+    {
+      "original": "base_datei",
+      "xchange": "overlay_datei"
```

Comment:
I do not argue against having this here, but question the name of the properties. Why not

```json
  "substFiles": [    {
      "base": "climate-timeseries.csv",
      "overlay": "mytimeseries_data.csv"
} ] }
```